gluten-core/src/main/scala/io/glutenproject/execution/BasicPhysicalOperatorTransformer.scala

```diff
@@ -392,7 +393,45 @@ object FilterHandler extends PredicateHelper {
           fileSourceScan,
           reuseSubquery,
           extraFilters = leftFilters)
+      case batchScan: BatchScanExec =>
+        val leftFilters = batchScan.scan match {
```

Comment:
I think "remaining" is more formal and less misleading than "left". Could you revise the naming here and at some other places? cc @rui-mo